R/summaries.R

```diff
@@ -275,33 +289,37 @@ setMethod("countDominantTaxa", signature = c(x = "SummarizedExperiment"),
             data <- data[rep(seq_len(nrow(data)), lengths(dominant_taxa_list)), ]
         }
         # Add dominant taxa to data
-        data$dominant_taxa <- dominant_taxa
+        colname <- "dominant_taxa"
+        data[[colname]] <- dominant_taxa
         # Gets an overview
-        .tally_col_data(data, group, name = "dominant_taxa")
+        .tally_col_data(data, group, digits, colname, name)
```

Comment:
This removes the problem of spaces in column  names? --> Good!

One suggestion:

Do not feed name parameter to this function; it makes things little bit harder to read.

Remove these modifications regarding the naming of column, instead, store the table to variable and then change the column name.
```

        data$dominant_taxa <- dominant_taxa
        tab <- .tally_col_data()
        # Change the column name to correct one
```